package.json

```diff
@@ -444,6 +444,30 @@
             {
                 "type": "python"
             }
+        ],
+        "languageModelTools": [
+            {
+                "name": "python_get_packages",
+                "displayName": "Get Python Packages",
+                "modelDescription": "Given a file path, finds the workspace and selected python environment for this file, and returns the list of installed packages.",
+                "toolReferenceName": "pythonGetPackages",
+                "tags": [
+                    "vscode_editing"
+                ],
+                "icon": "$(files)",
+                "inputSchema": {
+                    "type": "object",
+                    "properties": {
+                        "filePath": {
+                            "type": "string"
+                        }
+                    },
+                    "required": [
+                        "filePath"
+                    ]
+                },
+                "canBeReferencedInPrompt": true
```

Comment:
Should there be a `when` clause so this is only available when there's Python in the workspace?